chromium_src/third_party/blink/renderer/core/frame/navigator.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_FRAME_NAVIGATOR_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_FRAME_NAVIGATOR_H_
+
+#define BRAVE_NAVIGATOR_OVERRIDES \
+    private: \
+      Member<NavigatorBrave> brave_; \
+    public: \
+      NavigatorBrave* brave() { if (!brave_) { brave_ = MakeGarbageCollected<NavigatorBrave>(); } return brave_; }
```

Comment:
The only tiny suggestion I have is to move this into the implementation, since (i think) this will end up inlining the `MakeGarbageCollected` and `NavigatorBrave` constructors too, which can get big.  @bridiver would know better than me, but that was my understanding from the Chromium folks.

Reply:
ah- good call; that's definitely a possibility. In that case, I could easily implement this method in a .cc override

Reply:
I don't think this patch is needed at all. Please see `NavigatorGeolocation`